R/namespace.R

```diff
@@ -37,9 +37,14 @@ imported_s3_generics <- function(ns_imports) {
 
 is_s3_generic <- function(fun) {
   if (getRversion() >= "3.5.0") {
```

Comment:
Is the version requirement still necessary? It was introduced as a safeguard against a buggy implementation of `utils::isS3stdGeneric` in R 3.4.0

Reply:
Yup, I can use this new function for all versions, it looks like there's a test for 3.4.0, but not before that. That's probably fine since 3.3.0 and earlier is quite old at this point.